lib/rules/rules.json

```diff
@@ -131,6 +131,13 @@
         "alternative": "Array.prototype.slice()"
     },
 
+    "bind": {
+        "ruleName": "bind",
+        "compatible": true,
+        "alternative": "Function.prototype.bind()",
+        "ES6": true
```

Comment:
I don't think this is es6
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind#Specifications

Reply:
Right. Fixed.